INSTALLATION.md

```diff
@@ -198,36 +200,32 @@ While being logged into your administrator account, go to [http://localhost:3000
 Review the settings (if you want; you can change them later) and click "Save and continue" to complete
 setting up the dev server.
 
-### Configure Categories
+## Create a Post
 
-Before you try to create a post we need to configure categories!
-Go to `http://localhost:3000/categories/`
+You should then be able to create a post! There are character requirements for the
+body and title, and you are required at least one tag.

```

Comment:
I think you meant something along the lines of "and you have to add at least one tag", not that the poster is required to be tagged at least once :)

Reply:
Well, somebody before me, but I should have caught it. :-)